wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/SettingsDebugReportViewController.swift

```diff
@@ -0,0 +1,132 @@
+            title: Strings.TechnicalReport.sendReport.capitalized,
+            action: #selector(didTapSendReport)
+        )
+    }()
+
+    private lazy var shareReportButton: UIButton = {
+        return createButton(
+            title: Strings.TechnicalReport.shareReport.capitalized,
+            action: #selector(didTapShareReport)
+        )
+    }()
+
+    // MARK: - Life cycle
+
+    init(viewModel: SettingsDebugReportViewModelProtocol) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = SemanticColors.View.backgroundDefault
+        navigationItem.setDynamicFontLabel(title: Strings.TechnicalReportSection.title.capitalized)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    // MARK: - Setup
+
+    private func setupViews() {
+        view.addSubview(infoLabel)
+        view.addSubview(sendReportButton)
+        view.addSubview(shareReportButton)
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            infoLabel.topAnchor.constraint(equalTo: view.safeTopAnchor, constant: 20),
+            infoLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            infoLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+
+            shareReportButton.bottomAnchor.constraint(equalTo: view.safeBottomAnchor, constant: -30),
+            shareReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            shareReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            shareReportButton.heightAnchor.constraint(equalToConstant: 48),
+
+            sendReportButton.bottomAnchor.constraint(equalTo: shareReportButton.topAnchor, constant: -8),
+            sendReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            sendReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            sendReportButton.heightAnchor.constraint(equalToConstant: 48)
+        ])
+    }
+
+    // MARK: - Actions
+
+    @objc private func didTapSendReport() {
+        viewModel.sendReport()
+    }
+
+    @objc private func didTapShareReport() {
+        viewModel.shareReport()
+    }
+
+    // MARK: - Helpers
+
+    private func createButton(title: String, action: Selector) -> UIButton {
+        let button = ZMButton(
+            style: .secondaryTextButtonStyle,
+            cornerRadius: 16,
+            fontSpec: .buttonBigSemibold
+        )
+        button.setTitle(title, for: .normal)
+        button.addTarget(self, action: action, for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
```

Comment:
1. I would recommend create a new init for ZMButton so we can start using our new Font Styles instead of the old ones. 

```swift
    convenience init(
        style: ButtonStyle,
        cornerRadius: CGFloat,
        textStyle: WireTextStyle = .body1,
    ) {
        self.style = style
        self.font = .font(for: textStyle)
        self.adjustsFontForContentSizeCategory = true
        textTransform = .none
        layer.cornerRadius = cornerRadius
        contentEdgeInsets = UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16)

        applyStyle(style)
    }
```
    
2.  We would like to move away from using the #selector pattern. What we're using instead is UIAction. It can look something like this:

```swift
let action = UIAction { [weak self] _ in                                 
   self?.handleScrollToBottomTapped()
}
 button.addAction(action, for: .touchUpInside)
```

That allow us to remove the the @objc prefix before each method related to button actions

Reply:
1. Tried to do, but turned out to be too much work for this scope
2. done https://github.com/wireapp/wire-ios/pull/1658/commits/8c85d9ddbe3d4f96ae06fb91a9db48865e963819